karton/classifier/classifier.py

```diff
@@ -105,48 +161,52 @@ def process(self, task: Task) -> None:
             self.send_task(res)
             return
 
-        classification_tag = get_tag(sample_class)
-        self.log.info(
-            "Classified {!r} as {} and tag {}".format(
-                file_name.encode("utf8"), repr(sample_class), classification_tag
-            )
-        )
-
-        derived_headers = {
-            "type": "sample",
-            "stage": "recognized",
-            "quality": task.headers.get("quality", "high"),
-            "mime": sample_class["mime"],
-        }
-        if sample_class.get("kind") is not None:
-            derived_headers["kind"] = sample_class["kind"]
-        if sample_class.get("platform") is not None:
-            derived_headers["platform"] = sample_class["platform"]
-        if sample_class.get("extension") is not None:
-            derived_headers["extension"] = sample_class["extension"]
+        for sample_class in sample_classes:
 
-        derived_task = task.derive_task(derived_headers)
-
-        # pass the original tags to the next task
-        tags = [classification_tag]
-        if derived_task.has_payload("tags"):
-            tags += derived_task.get_payload("tags")
-            derived_task.remove_payload("tags")
-
-        derived_task.add_payload("tags", tags)
-
-        # if present the magic description is added as a playload
-        if "magic" in sample_class:
-            derived_task.add_payload("magic", sample_class["magic"])
-
-        # add a sha256 digest in the outgoing task if there
-        # isn't one in the incoming task
-        if "sha256" not in derived_task.payload["sample"].metadata:
-            derived_task.payload["sample"].metadata["sha256"] = sha256(
-                cast(bytes, sample.content)
-            ).hexdigest()
+            classification_tag = get_tag(sample_class)
+            self.log.info(
+                "Classified {!r} as {} and tag {}".format(
+                    file_name.encode("utf8"), repr(sample_class), classification_tag
+                )
+            )
 
-        self.send_task(derived_task)
+            derived_headers = {
+                "type": "sample",
+                "stage": "recognized",
+                "kind": sample_class["kind"],
```

Comment:
This previously was
```
        if sample_class.get("kind") is not None:
            derived_headers["kind"] = sample_class["kind"]
```

But since `kind` must be always returned (its even checked by the `get_tag` method) we can assume that it'll always be there